components/ApplyToHostBtn.js

```diff
@@ -11,6 +10,8 @@ import StyledButton from './StyledButton';
 import StyledTooltip from './StyledTooltip';
 import { withUser } from './UserProvider';
 
+import { Question } from '@styled-icons/fa-solid/Question';
```

Comment:
Why destructured imports from libraries are coming after locally destructured imports?

Reply:
I'm proposing to group images together wherever they're coming from. Everything coming from @styled-icons is considered an image.